frontend/src/js/route-config/route-config.js

```diff
@@ -183,6 +183,15 @@
             authenticate: true
         };
 
+        var all_challenges = {
+            name: 'all-challenges',
+            url: "/challenges/list",
+            templateUrl: baseUrl + "/web/challenge/list_all_challenges.html",
+            controller: 'ChallengeListAllCtrl',
```

Comment:
Use 'challengeListCtrl' here

Reply:
@aka-jain I think we can't use it here as it also sends a parameter of `user token` in it but we don't have the user token.